frameworks/helloworld/src/main/dist/examples/taskcfg.yml

```diff
@@ -25,7 +22,7 @@ pods:
         cmd: "echo world-$POD_INSTANCE_INDEX >> world-container-path/$OUTPUT_FILENAME && sleep $SLEEP_DURATION"
         cpus: {{WORLD_CPUS}}
         memory: {{WORLD_MEM}}
-        volumes:
-          - path: "world-container-path"
-            type: ROOT
-            size: {{WORLD_DISK}}
+        volume:
+          path: "world-container-path"
```

Comment:
Does this prevent (or require an API change) to supporting multiple volumes in the future?